app/src/main/java/com/github/se/cyrcle/ui/profile/ProfileScreen.kt

```diff
@@ -271,9 +277,9 @@ private fun ProfileImage(
 }
 
 @Composable
-private fun FavoriteParkingsSection(
-    favoriteParkings: List<String>,
-    onFavoritesUpdated: (List<String>) -> Unit
+fun FavoriteParkingsSection(
```

Comment:
Maybe you should let the function private since it will be only useful for this screen.